packages/progress-indicator/test/lion-progress-indicator.test.js

```diff
@@ -1,23 +1,155 @@
-import { expect, fixture } from '@open-wc/testing';
 import { html } from '@lion/core';
+import { expect, fixture as _fixture } from '@open-wc/testing';
 
 import '@lion/progress-indicator/define';
 
+/**
+ * @typedef {import('../src/LionProgressIndicator').LionProgressIndicator} LionProgressIndicator
+ * @typedef {import('@lion/core').TemplateResult} TemplateResult
+ */
+
+const fixture = /** @type {(arg: TemplateResult) => Promise<LionProgressIndicator>} */ (_fixture);
+
 describe('lion-progress-indicator', () => {
-  describe('Accessibility', () => {
-    it('adds a label', async () => {
+  describe('indeterminate', async () => {
+    it('is indeterminate when has no value attribute', async () => {
       const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
-      expect(el.getAttribute('aria-label')).to.equal('Loading');
+      expect(el.indeterminate).to.be.true;
     });
 
-    it('sets the right role', async () => {
+    it('adds a label by default', async () => {
       const el = await fixture(html` <lion-progress-indicator></lion-progress-indicator> `);
-      expect(el.getAttribute('role')).to.equal('status');
+      await el.localizeNamespacesLoaded;
+      expect(el.getAttribute('aria-label')).to.equal('Loading');
     });
 
-    it('sets aria-live to "polite"', async () => {
+    it('can override a label with "aria-label"', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator aria-label="foo"></lion-progress-indicator> `,
+      );
+      await el.localizeNamespacesLoaded;
+      expect(el.getAttribute('aria-label')).to.equal('foo');
+      el.setAttribute('aria-label', 'bar');
+      expect(el.getAttribute('aria-label')).to.equal('bar');
+      el.removeAttribute('aria-label');
+      await el.updateComplete;
+      expect(el.getAttribute('aria-label')).to.equal('Loading');
+    });
+
+    it('can override a label with "aria-labelledby"', async () => {
+      const el = await fixture(
+        html` <lion-progress-indicator aria-labelledby="foo-id"></lion-progress-indicator> `,
+      );
+      await el.localizeNamespacesLoaded;
+      expect(el.getAttribute('aria-labelledby')).to.equal('foo-id');
+      // expect(el.hasAttribute('aria-label')).to.be.false;
+      el.setAttribute('aria-labelledby', 'bar-id');
+      expect(el.getAttribute('aria-labelledby')).to.equal('bar-id');
+      expect(el.hasAttribute('aria-label')).to.be.false;
+      el.removeAttribute('aria-labelledby');
+      await el.updateComplete;
+      expect(el.hasAttribute('aria-labelledby')).to.be.false;
+      expect(el.getAttribute('aria-label')).to.equal('Loading');
+    });
+  });
+
+  describe('determinate', async () => {
+    it('is determinate when has value attribute', async () => {
```

Comment:
```suggestion
    it('is determinate when has a value', async () => {
```
(because .value prop is SSOT, and attr is a reflection of that or means to set SSOT)